commands/commands.js

```diff
@@ -0,0 +1,28 @@
+import chalk from "chalk";
+import promptSync from "prompt-sync";
+import { constantValues } from "../constants/consts.js";
+import { shortcuts } from "../constants/shortcut.js";
+import { listCategories } from "../api/category.js";
+
+export const prompt = promptSync();
+
+export const askName = () => {
+    console.log("\n \n " + chalk.green(constantValues.title) + "\n \n" + chalk.bgGreen(constantValues.description) + "\n\n" + constantValues.jokeCategories + chalk.red(shortcuts.see_joke_categories) + "\n" + constantValues.backToStart + chalk.red(shortcuts.go_back_to_start) + "\n" + constantValues.exitProgram + chalk.red(shortcuts.exit_project) + "\n");
```

Comment:
You can keep this into a separate function named printBanner or something like that as this can be reuse